src/components/tests/QuickOpenModal.spec.js

```diff
@@ -101,4 +109,55 @@ describe("QuickOpenModal", () => {
     wrapper.setProps({ enabled: true });
     expect(wrapper).toMatchSnapshot();
   });
+
+  test("basic source search", () => {
+    const { wrapper } = generateModal(
+      {
+        enabled: true,
+        symbols: {
+          functions: [],
+          variables: []
+        }
+      },
+      "mount"
+    );
+    wrapper.find("input").simulate("change", { target: { value: "somefil" } });
+    expect(filter).toHaveBeenCalled();
+  });
+
+  test("basic gotoSource search", () => {
+    const { wrapper } = generateModal(
+      {
+        enabled: true,
+        searchType: "gotoSource",
+        symbols: {
+          functions: [],
+          variables: []
+        }
+      },
+      "mount"
+    );
+    wrapper
+      .find("input")
+      .simulate("change", { target: { value: "somefil:33" } });
+    expect(filter).toHaveBeenCalled();
```

Comment:
can we be more specific w/ the params passed to filter?

For instance i would guess filter would not be called if we include a `:##` param

Reply:
`.toHaveBeenCalledWith(...)` over `toHaveBeenCalled()`?

Reply:
`toHaveBeenCalledWith` would be good because we can assert the params that are passed as well